src/style/sections/_section-openmap.scss

```diff
@@ -38,16 +42,32 @@
         font-weight: bold;
         text-transform: uppercase;
         color: $color-blush;
-        
-        &:hover {
-            font-size: $font-large;
-            transition-duration: 0.5s;
-        }
     }
 
     &__point {
         font-size: $font-large;
         color: $color-blush;
     }
 
+    &__close {
+        display: none;
+        width: 100%;
+        padding-top: 10px;
+        position: absolute;
+        bottom: -35px;
```

Comment:
it's usually not very good to use values below zero. Is it possible to refactor this?